File: 8_kyu/Quadrants.rs
// Returns number of quadrant of the point (x,y).
fn quadrant(x: i32, y: i32) -> i32 {
    match (x < 0, y < 0) {
        (false, false) => 1,
        (true, false) => 2,
        (true, true) => 3,
        (false, true) => 4,
    }
}

File: 8_kyu/Quadrants.py
def quadrant(x :float, y :float) -> int:
    """
    Quadrant 1-4 of the point (x,y) in the plane.
    """
    if x >= 0:
        return 1 if y >= 0 else 4
    else:
        return 2 if y > 0 else 3
